docs/make.jl

```diff
@@ -3,15 +3,21 @@
 # Licensed under the MIT license. See LICENSE file in the project root for details.
 #
 
-using Documenter, FMI
+import Pkg; Pkg.develop(path=joinpath(@__DIR__,"../../FMI.jl"))
+using Documenter, Plots, JLD2, DataFrames, CSV, MAT, FMI, FMIImport, FMICore
 using Documenter: GitHubActions
 
 makedocs(sitename="FMI.jl",
          format = Documenter.HTML(
             collapselevel = 1,
             sidebar_sitename = false,
-            edit_link = nothing
+            edit_link = nothing,
+            size_threshold_ignore = ["library.md","deprecated.md","fmi2_lowlevel_librarys.md","fmi3_lowlevel_librarys.md"]
```

Comment:
necessary, will be removed as soon as library.md and fmiX_lowlevel_librarys.md are spilt just as fmi_lowlevel_library_types, _functions, _constants